### PR TITLE
PID for Tron Guy Labs M122/3270 USB keyboard

### PR DESCRIPTION
**Pull request by jmaynard**
Requesting PID 1209/3270 for a controller board for the IBM Model M 122-key keyboard. This isn't a converter, but replaces the internal controller entirely. It uses the open-source QMK firmware. My code is in the QMK repository I linked, in the keyboards/tronguylabs directory. The schematic and Gerber files for a carrier board that connects the matrix connectors to the BlackPill MCU can be found at https://github.com/jmaynard/tronguylabs-m122-blackpill ; those files are licensed under the Solderpad Hardware License v2.1.

### CODE REVIEW

**Review by tannewt — CHANGES_REQUESTED** [review]
Please include the modifiable EasyEDA files from here: https://easyeda.com/dillon/Backup_Your_EasyEDA_Project_Locally-JrecamWv5

Please also link to the hardware files from here and link to the QMK files from the readme of the hardware source. That will make it easier to find the hardware from the pid.codes website.

**Comment by jmaynard** [comment]
> Please include the modifiable EasyEDA files from here: https://easyeda.com/dillon/Backup_Your_EasyEDA_Project_Locally-JrecamWv5
> 
> Please also link to the hardware files from here and link to the QMK files from the readme of the hardware source. That will make it easier to find the hardware from the pid.codes website.

I have a pull request in for the QMK sources that's waiting on approval of this PR to get merged into their repository. We have a chicken and egg situation. Shall I link to my QMK fork, or...?

**Comment by jmaynard** [comment]
> > Please include the modifiable EasyEDA files from here: https://easyeda.com/dillon/Backup_Your_EasyEDA_Project_Locally-JrecamWv5
> > Please also link to the hardware files from here and link to the QMK files from the readme of the hardware source. That will make it easier to find the hardware from the pid.codes website.
> 
> I have a pull request in for the QMK sources that's waiting on approval of this PR to get merged into their repository. We have a chicken and egg situation. Shall I link to my QMK fork, or...?

For now, I've linked to my QMK fork. I also added the EasyEDA files to the hardware repository, and linked my QMK fork from my hardware repository; once this PR and the QMK PR are approved, I'l move the link on the hardware to the proper place in the QMK tree.

**Comment by jmaynard** [comment]
The code has now been merged to the QMK tree, and I've updated the links from the hardware repository and this PR to point to it.

**Review by tannewt — APPROVED** [review]
Looks good. Thanks for the update

**Pull request merged**
